app/controllers/api/v1/accounts_controller.rb

```diff
@@ -55,7 +55,7 @@ def account_params
   end
 
   def check_signup_enabled
-    raise ActionController::RoutingError, 'Not Found' if GlobalConfig.get_value('ENABLE_ACCOUNT_SIGNUP') == 'false'
+    raise ActionController::RoutingError, 'Not Found' if GlobalConfigService.load('ENABLE_ACCOUNT_SIGNUP', '') == 'false'
```

Comment:
The default should be false instead of empty string, right? Other if the env is not present, then this condition would yield false, allowing the signup